src/Interpolation/CIC.h

```diff
@@ -119,6 +119,41 @@ namespace ippl {
             const std::index_sequence<GatherPoint...>&, const View& view,
             const Vector<T, View::rank>& wlo, const Vector<T, View::rank>& whi,
             const Vector<IndexType, View::rank>& args);
+
+        template <unsigned long ScatterPoint, unsigned long... Index, typename T, unsigned Dim,
+                  typename IndexType>
+        KOKKOS_INLINE_FUNCTION constexpr int ZigzagScatterToPoint(
+            const std::index_sequence<Index...>&,
+            const typename ippl::detail::ViewType<ippl::Vector<T, 3>, Dim>::view_type& view,
```

Comment:
If the `3` is correct, there should be some documentation to explain why it's 3 and not `Dim`

Reply:
it's wrong, thanks for poynting out